### ISSUE TITLE
Installer Not Working

### ISSUE THREAD

**Issue by DDynamic** [PufferPanel/Tools#5]
Hi, I was trying to install PufferPanel 0.8.0 Beta (just wanted to test it!). The cURL installer returned some issues during the MySQL section, so I copied and pasted the installer located [here](https://raw.githubusercontent.com/PufferPanel/Tools/master/installer/installer). I ran it using ./installer, the file I had made for it. I get this:

```
THIS VERSION OF PUFFERPANEL CANNOT BE INSTALLED WITH THIS INSTALLER 
An error occured while installing, halting
```

You can view my entire installation process log [here](http://pastebin.com/xtTxUV53). I followed the documentation located [here](http://www.pufferpanel.com/v0.8.0/docs).

Cheers & thank you in advance.

**Comment by DaneEveritt** [comment]
Pretty sure you just need to run it with the `-d` flag, otherwise it tries to install the last released version, which is 0.7.6.1, which the installer is not compatible with.

```
-----
Configuration options complete, beginning installation process
-----
Cloning PufferPanel to /tmp/tmp.vBVgqCtNEZ
Cloning into '/tmp/tmp.vBVgqCtNEZ'...
remote: Counting objects: 14483, done.
remote: Compressing objects: 100% (70/70), done.
remote: Total 14483 (delta 28), reused 0 (delta 0), pack-reused 14407
Receiving objects: 100% (14483/14483), 21.27 MiB | 1.32 MiB/s, done.
Resolving deltas: 100% (8953/8953), done.
-----
Checking out 0.7.6.1-beta
Note: checking out 'tags/0.7.6.1-beta'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b new_branch_name

HEAD is now at 23ce687... Bump version to 0.7.6.1 Beta
-----
```

**Comment by DDynamic** [comment]
**Edit:** That seemed to do the trick.
